packages/@okta/i18n/src/properties/login.properties

```diff
@@ -43,6 +43,7 @@ error.unsupported.cors = Unsupported browser - missing CORS support
 error.unsupported.localStorage = Unsupported browser - missing localStorage support
 error.enabled.cors = There was an error sending the request - have you enabled CORS?
 error.expired.session = Your session has expired. Please try to log in again.
```

Comment:
I guess this was lost in the previous comment, I'm 100% with changing this to `Please try to sign in again.`. Also good if you prefer to do it in another jira/PR since that involves selenium tests as well

Reply:
yes, sorry. I thought i would leave this as is for now.. but maybe it is better to change this to `sign in` in this PR itself.